### PR TITLE
Show users tips to avoid scams on account creation

### PR DESCRIPTION
**Pull request by piyalbasu**
Screen Shot 2020-10-06 at 10 16 18 PM

### CODE REVIEW

File: extension/src/popup/views/FullscreenSuccessMessage.tsx [hunk 1]
```diff
@@ -50,27 +60,78 @@ const IlloContainerEl = styled.div`
...
+  margin: 0;
+`;
+
+const MessageListEl = styled.ul`
+  padding: 0 1.3rem;
+
+  li {
+    margin-top: 1rem;
+  }
+`;
```
Comment (piyalbasu) [inline extension/src/popup/views/FullscreenSuccessMessage.tsx:95]: this alert message layout is very *similar* to what we do in `<WarningMessage />`, but just different enough that it doesn't really make sense to share any code. Kinda annoying.

This whole app would benefit from a design based on a system. Something I'll push for with the design team for v2

File: extension/src/popup/App.tsx [hunk 2]
```diff
@@ -23,10 +23,14 @@ const GlobalStyle = createGlobalStyle`
...
+    height: ${POPUP_HEIGHT}px;
     width: ${POPUP_WIDTH}px;
   }
-  body, html, #root {
-    height: ${POPUP_HEIGHT}px;
+
+  #root {
+    display: flex;
+    flex-flow: column;
+    height: 100%;
```
Comment (piyalbasu) [inline extension/src/popup/App.tsx:33]: there were some issues with larger screen sizes. The blue background would stop abruptly in the middle of the page. This helps fix that

File: extension/src/popup/views/FullscreenSuccessMessage.tsx [hunk 3]
```diff
@@ -25,7 +29,7 @@ const WrapperEl = styled.div`
   background: ${COLOR_PALETTE.primary};
   display: flex;
   flex-direction: column;
-  height: calc(100vh - ${HEADER_HEIGHT}px);
+  flex-grow: 1;
```
Comment (piyalbasu) [inline extension/src/popup/views/FullscreenSuccessMessage.tsx:32]: this makes the div extend to the bottom of the page

File: extension/src/popup/constants/styles.tsx [hunk 4]
```diff
@@ -35,3 +35,5 @@ export const FONT_WEIGHT = {
   normal: "400",
   bold: "600",
 };
+
+export const ROUNDED_CORNERS = "1.25rem";
```
Comment (jeesunikim) [inline extension/src/popup/constants/styles.tsx:39]: TY!

**Review by jeesunikim — APPROVED** [review]

**Pull request merged**